napari/_qt/qt_viewer.py

```diff
@@ -367,23 +369,11 @@ def screenshot(self, path=None):
             imsave(path, QImg2array(img))  # scikit-image imsave method
         return QImg2array(img)
 
-    def _screenshot_dialog(self):
+    def _screenshot_dialog(self, *, default_name=""):
```

Comment:
```suggestion
    def _screenshot_dialog(self):
```
`deafult_name` can be cut